deployment/keystone/deploy.go

```diff
@@ -423,45 +423,65 @@ func ConfigureOCR3Contract(env *deployment.Environment, chainSel uint64, dons []
 	return nil
 }
 
-func ConfigureOCR3ContractFromJD(env *deployment.Environment, chainSel uint64, nodeIDs []string, addrBook deployment.AddressBook, cfg *OracleConfigWithSecrets) error {
-	registryChain, ok := env.Chains[chainSel]
+type ConfigureOCR3Resp struct {
+	Orc2drOracleConfig
```

Comment:
what does this name mean? 

//nit could you leave a comment or change to a more intuitive name?

Reply:
I guess this means OCR2 Dry Run Oracle Config?